ZenCompetition.tex

```diff
@@ -107,7 +107,7 @@
 
 \item \textbf{Performance:} Before proceeding into the obstacle course, the robot will be judged on a 90 second animation intended to hype up the crowd. At each save-point through the course another 90 second animation will be required. 
 
-\item \textbf{Eligibility:} High school and Jr High school students affiliated with public schools or not-for-profit private schools. Pay-to-play organizations are explicitly forbidden, all students must be able to join teams without a per student cost borne by the students or their families. 
+\item \textbf{Eligibility:} High school and Jr High school students affiliated with public schools or not-for-profit private schools or personal households. Pay-to-play organizations are explicitly forbidden, all students must be able to join teams without a per student cost borne by the students or their families. 
```

Comment:
"...or home schooled students provided they maintain a full academic course-load in addition to the competition. "